.gitignore

```diff
@@ -26,10 +26,13 @@ config/secrets.yml
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
```

Comment:
I already included .DS_Store into my ignore folder. I tried to run the command git rm .DS_Store and was told that there are no such files.